src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageExtensions.cs

```diff
@@ -67,9 +68,20 @@ public static class LoggerConfigurationAzureTableStorageExtensions
             if (loggerConfiguration == null) throw new ArgumentNullException("loggerConfiguration");
             if (storageAccount == null) throw new ArgumentNullException("storageAccount");
 
-            var sink = writeInBatches ?
-                (ILogEventSink)new AzureBatchingTableStorageSink(storageAccount, formatProvider,  batchPostingLimit ?? DefaultBatchPostingLimit, period ?? DefaultPeriod, storageTableName, keyGenerator) :
-                new AzureTableStorageSink(storageAccount, formatProvider, storageTableName, keyGenerator);
+			ILogEventSink sink;
+
+			try
+			{
+				sink = writeInBatches ?
+					(ILogEventSink)new AzureBatchingTableStorageSink(storageAccount, formatProvider, batchPostingLimit ?? DefaultBatchPostingLimit, period ?? DefaultPeriod, storageTableName, keyGenerator) :
+					new AzureTableStorageSink(storageAccount, formatProvider, storageTableName, keyGenerator);
+			}
+
+			catch (Exception ex)
+			{
+				Debugging.SelfLog.WriteLine("Error configuring AzureTableStorage: {0}", ex);
+				sink = new NullSink();
```

Comment:
Nice... I copied `NullSink` from the File sink project. That's a handy shortcut.